src/Console/src/Configurator/Configurator.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Spiral\Console\Configurator;
+
+use Spiral\Console\Command;
+
+/**
+ * @internal
+ */
+final class Configurator implements ConfiguratorInterface
```

Comment:
Could you get rid of the interface here? Is it necessary?

Reply:
Removed interface implementation here